src/services/apiService.ts

```diff
@@ -0,0 +1,58 @@
+import { IMinimalRepository, OrganizationSimple, ICommitResponse } from '../models'
+
+const username: string = 'aaronabramov'; // will dynamically change the user later (fron state)
+const baseUrl: string = `https://api.github.com/users/${username}`;
```

Comment:
This will probably change once you incorporate TanStack but I wouldn't advise having the username as part of the baseUrl 😅 
I suggest having ` baseUrl + username + '/repos' ` whenever you query for a specific user's Github details

Reply:
Yeah, I agree. Most of the queries I had included the userName, but then the last one came. So I'll have to change it. Thanks!